web_notify/models/res_users.py

```diff
@@ -80,6 +85,7 @@ def _notify_channel(
             "message": message,
             "title": title,
             "sticky": sticky,
+            "allow_native_notifications": self.allow_native_notifications,
```

Comment:
this makes all the comparisons in tests fail. You need to adapt https://github.com/OCA/web/blob/f56f71155f23ce07bad57b3a7f9d812412ff5e3a/web_notify/tests/test_res_users.py#L24 and below